README.rst

```diff
@@ -44,15 +44,15 @@ Requirements
 ------------
 
 * Python 3.6+
-* psycopg 2
+* psycopg 3
 
 
 Installation
 ------------
 
 .. code-block:: bash
 
-    ; If you do not have psycopg2 yet, install it as `psycopg2` or `psycopg2-binary`. 
+    ; If you do not have psycopg3 yet, install it as `psycopg` or `psycopg[binary]`.
     ; You may also want to install `envbox` to get PG connection settings from .env files.
     ; Your distribution may require issuing `pip3` command instead of plain `pip`.
     $ pip install psycopg2-binary envbox
```

Comment:
Old command is used.

Reply:
Didn't get what exactly has to be changed here, honestly. Could you give an example?

Reply:
I'd expect something like `pip install psycopg[binary] envbox`

Reply:
You was right, fixed this in new commit